democracylab/emails.py

```diff
@@ -201,6 +201,7 @@ def send_to_project_volunteer(volunteer_relation, subject, template):
         subject=subject,
         from_email=settings.EMAIL_VOLUNTEER_ACCT['from_name'],
         to=[volunteer_relation.volunteer.email],
+        cc=co_owner_emails,
```

Comment:
We need to include the project owner in this list as well.  You can get the project owner's email with
volunteer_relation.project.project_creator.email